rules/xcodeproj.bzl

```diff
@@ -498,6 +493,8 @@ def _populate_xcodeproj_targets_and_schemes(ctx, targets, src_dot_dots, all_tran
 
         target_settings = {
             "PRODUCT_NAME": target_name,
+            "ARCHS": "$(ARCHS_STANDARD)",
+            "EXCLUDED_ARCHS[sdk=iphonesimulator*]": "arm64",
```

Comment:
curious the effect of this on silicon macbook

Reply:
I'd presume we'd need to update this if we need to build for arm64 sims, but we can deal with that later.